site/_posts/2023-03-09-react-example-application.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: blog-post
+title: Using the FusionAuth React SDK in a React application
+description: In this guide, we'll showcase how to use the FusionAuth React SDK to implement the login, logout, and registration flows in a React application.
+author: Colin Frick
+category: blog
+image: blogs/react-example-application/react-example-application.png
+tags: tutorial tutorial-react tutorial-typescript react typescript oauth javascript
+excerpt_separator: "<!--more-->"
+---
+
+The React SDK from FusionAuth enables developers to implement the login, logout, and registration flows in a React application with ease.
+
+<!--more-->
+
+To show the React SDK in action, we will be using the FusionDesk application. FusionDesk is a simple help desk example application that allows users to create tickets and view them. The application is built using React and Node.js. It uses a server backend built using Ts.Ed, Node.js, and Typescript. The FusionDesk application is available on GitHub.
```

Comment:
"available on GitHub" should be linked to the GH repo. That should be transferred to the FusionAuth org (or just give me read rights and I can move it).

Reply:
Ah, I see you added it below. I'd remove "The FusionDesk application is available on GitHub." because you show that in a few paragraphs.